NLog.Web.Tests/LayoutRenderers/AspNetSessionValueLayoutRendererTests.cs

```diff
@@ -23,6 +24,7 @@ public AspNetSessionValueLayoutRendererTests()
 
         public void SetUp()
         {
+            ConfigurationItemFactory.Default.LayoutRenderers.RegisterDefinition("aspnet-session", typeof(AspNetSessionValueLayoutRenderer));
```

Comment:
@304NotModified here you go - it's in petemounce/NLog.Web@2ba53a1921cd675f2885e4413c10a08d02cb2433. I branched from up-to-date master, made the code change, and have posted the log into that commit message.